src/utils/transaction-guards.ts

```diff
@@ -101,6 +105,10 @@ export const isCancellationTxInfo = (value: TransactionInfo): value is Cancellat
   return isCustomTxInfo(value) && value.isCancellation
 }
 
+export const hasSafeAppInfo = <T extends TransactionSummary>(value: T): value is T & { safeAppInfo: SafeAppInfo } => {
+  return !!value.safeAppInfo
+}
```

Comment:
Why do we need this guard instead of just checking `if (tx.safeAppInfo)` like before? The method signature looks more complex.

Reply:
I was following the pattern of guards. In hindsight, I agree with you. I've removed it.